lib/kafka_ex/default_server.ex

```diff
@@ -0,0 +1,309 @@
+defmodule KafkaEx.DefaultServer do
```

Comment:
Which version is this for?  I guess what I'm saying is why not just call it KafkaEx.Impl0p9 or whatever version it implements?